### DISCUSSION TITLE
[mpvnet] 完善mpv.net_CM的mpv.conf编辑器内容，以供参考

### DISCUSSION THREAD

**Discussion by dyphire** [hooke007/mpv_PlayKit#40] [libmpv]
对mpv.net_CM的mpv.conf图形编辑器增加部分可调整选项，尽可能覆盖更多的需求场景。由于不便于提交PR，在此提供下文件内容，以供参考

[dyphire/mpv.net/editor_conf.txt](https://github.com/dyphire/mpv.net/blob/main/src/Resources/editor_conf.txt)

**Comment by hooke007** [comment]
很多选项没加是因为

我觉得没必要（
当初汉化也是为了初级用户，删减了一些选择防止误操作产生奇怪的bug
基本不用的或固定的选择就根本没加，不然每个scale算法写全就能各自占几页。。。

**Reply by dyphire** [reply]
精简导致缺失项很多，作为图形编辑器的初衷应该是尽可能地提供配置组参数以外的高频选项。很多默认项和基本用不到的高级选项确实没必要体现，我也没放进去，基于个人需求考虑增加了使用场景的覆盖率，尽可能地降低手动编辑主配置mpv.conf的概率

**Reply by dyphire** [reply]
不过初衷是考虑到一般用户的话可以理解，我的修改版就当给有其他需求的多一个选择吧

**Comment by dyphire** [comment]
> https://github.com/hooke007/mpv.net_CM/blob/599c70cbc0ea4405ae64e4d04a40960bfc534bf1/src/Resources/input.conf.txt#L133
```
_                 set sub-delay 0; set sub-pos 0; set sub-scale 0       #menu: 字幕 > 恢复初始
```
这里有个错误，修复示例如下：
```
_                 set sub-delay 0; set sub-pos 100; set sub-scale 1.0   #menu: 字幕 > 恢复初始
```
